docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CREATE_REPLICATION_SLOT statement [YSQL]
+headerTitle: CREATE_REPLICATION_SLOT
+linkTitle: CREATE_REPLICATION_SLOT
+description: Use the CREATE_REPLICATION_SLOT statement to create a replication slot.
+menu:
+  preview:
+    identifier: streaming_create_repl_slot
+    parent: statements
+type: docs
+---
+
+## Synopsis
+
+Use the `CREATE_REPLICATION_SLOT` statement to create a replication slot.
+
+## Syntax
+
+{{%ebnf%}}
+  create_replication_slot
+{{%/ebnf%}}
+
+## Semantics
+
+### *slot_name*
+
+The name of the replication slot. The name of the replication slot must be **unique across all databases**.
+
+### *output_plugin*
+
+The name of the output plugin used for logical decoding.
+
+### NOEXPORT_SNAPSHOT / USE_SNAPSHOT
+
+Decides what to do with the snapshot created during logical slot initialization.
+
+`USE_SNAPSHOT` will use the snapshot for the current transaction executing the command. This option must be used in a transaction, and `CREATE_REPLICATION_SLOT` must be the first command run in that transaction.
```

Comment:
The semantics of USE_SNAPSHOT in our case is that an initial snapshot will be supported for this stream - "will use the snapshot for the current transaction executing the command" does not hold for YB

Reply:
Doc from yb-admin https://docs.yugabyte.com/preview/admin/yb-admin/#change-data-capture-cdc-commands

Reply:
Done, thank you